src/traces/parcoords/lines.js

```diff
@@ -396,9 +397,11 @@ module.exports = function(canvasGL, d) {
             ];
         }
 
-        var mask = Array.apply(null, new Array(maskHeight * channelCount)).map(function() {
-            return 255;
-        });
+        var mask = [];
+        for(var i = 0; i < maskHeight * channelCount; i++) {
```

Comment:
Having `var mask = new Array(maskHeight + channelCount);` would be even better :smirk:

Reply:
Good call. The arrays are pre-allocated in https://github.com/plotly/plotly.js/pull/3966/commits/75f692dda8b6905d6399cb05fc09ff9ab63fff56.